packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox/gux-form-field-checkbox.tsx

```diff
@@ -48,6 +57,15 @@ export class GuxFormFieldCheckbox {
   @Prop()
   hasGroupDisabled: boolean = false;
 
+  @Watch('hasError')
+  onHasError(hasError: boolean): void {
+    if (hasError) {
+      this.input.setAttribute('aria-invalid', 'true');
+    } else {
+      this.input.removeAttribute('aria-invalid');
+    }
+  }
+
```

Comment:
Linked to previous comment: This is where we might have a watch 'hasGroupError' too.